src/sass/file-preview-card/stories/file-preview-card.stories.js

```diff
@@ -0,0 +1,238 @@
+            </div>
+        </div>
+    </div>
+`;
+
+export const landscapeImage = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-landscape-pic.jpg" alt="Image">
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const video = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg" alt="Image">
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <svg class="icon icon--12 file-preview-card__video-icon" aria-hidden="true">
+                        <use href="#icon-play-16"></use>
+                    </svg>
+                    <span class="file-preview-card__label-text">0:21</span>
+                </button>
+            </div>
+        </div>
+    </div>
+`;
+
+export const videoRTL = () => `
+    <div style="height: 168px; width: 168px;">
+        <div class="file-preview-card" dir="rtl">
+            <div class="file-preview-card__center">
+                <img src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg" alt="Image">
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg class="icon icon--16" height="16" width="16" aria-hidden="true">
+                        <use href="#icon-overflow-vertical-16"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <svg class="icon icon--12 file-preview-card__video-icon" aria-hidden="true">
+                        <use href="#icon-play-16"></use>
+                    </svg>
+                    <span class="file-preview-card__label-text">0:21</span>
```

Comment:
Similar feedback here, we need to convey timestamp as part of aria-label.

Reply:
I would remove the aria-label from button so we at least have the button label matching the visual label (0:21) for voice users.